octobot_trading/personal_data/orders/order_util.py

```diff
@@ -365,15 +357,15 @@ async def get_order_size_portfolio_percent(exchange_manager, order_amount, side,
                                  portfolio_type=commons_constants.PORTFOLIO_TOTAL)
     if exchange_manager.is_future:
         # TODO check inverse
-        if market_quantity is constants.ZERO:
+        if market_quantity == constants.ZERO:
             return constants.ZERO
         return min(order_amount / market_quantity, constants.ONE) * constants.ONE_HUNDRED
     if side is enums.TradeOrderSide.SELL:
-        if current_symbol_holding is constants.ZERO:
+        if current_symbol_holding == constants.ZERO:
```

Comment:
?

Reply:
if current_symbol_holdeing is Decimal("0") it requires == instead of is or it will be false
see screenshot
![image](https://user-images.githubusercontent.com/32987678/206651361-d07bcb89-20c7-4de4-bcd8-7ac3581845d6.png)

Reply:
Right, thanks 👍